infra/config/validation.yaml

```
@@ -58,9 +58,12 @@ allowed_keywords:
   - name: test
     description: Add or update tests
     requires_scope: true
-    scopes: [ Unit, Integration, E2E ]
+    scopes: [ Unit, Integration, E2E, Benchmark ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)